test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1

```diff
@@ -16,4 +16,19 @@ Describe 'ConvertTo-Json' -tags "CI" {
         $jsonFormat | Should Match '"TestValue2": 78910'
         $jsonFormat | Should Match '"TestValue3": 99999'
     }
+
+	It "StopProcessing should succeed" {
+        $tmpFile = Join-Path $TestDrive "test.txt"
+        Set-Content -Path $tmpFile -Value "hello"
+        $ps = [PowerShell]::Create()
+        $null = $ps.AddCommand("Get-Content")
```

Comment:
I've a minor concern that if we end up changing the behavior of `Get-Content`, This test will be a false pass. Perhaps it would be better to generate a deep object on the fly that we know will always be deep?

```powershell
$ps = [PowerShell]::Create()
$null = $ps.AddScript({
    $obj = [PSCustomObject]@{P1 = ''; P2 = ''; P3 = ''; P4 = ''; P5 = ''; P6 = ''}
    $obj.P1 = $obj.P2 = $obj.P3 = $obj.P4 = $obj.P5 = $obj.P6 = $obj
    1..100 | Foreach-Object { $obj } | ConvertTo-Json -Depth 10
})
$handler = $ps.BeginInvoke()
```